src/utils/hooks/useAutoLogout.js

```diff
@@ -0,0 +1,30 @@
+import { getLogoutUrl } from 'components/auth/provider/component';
+import { decodeJwt } from 'oidc-spa';
+import { OIDC } from 'utils/constants';
+import { listenActivity } from 'utils/events';
+import { environment } from '../read-env-vars';
+
+const { AUTH_TYPE } = environment;
+
+export const useAutoLogout = ({ oidc }) => {
```

Comment:
Ce n'est pas un hook car ça n'utilise pas de hook à l'intérieur

Reply:
I fix that, thank you